### DISCUSSION TITLE
95 renLuna as WWT on eth wallet

### DISCUSSION THREAD

**Discussion by anonintern** [wormhole-foundation/wormhole#307] [Developer Q&A] [answered]
about 11 days ago sent renLuna from SOL to ETH and still shows in my wallet as WWT. is there anyway to unwrap this?

**Comment by leoluk** [comment, marked as answer]
You now own Wormhole-wrapped renLuna on ETH. You'd need to find a liquid market to swap this to something else, or send it back.

**Reply by anonintern** [reply]
where can i go to send the WWT back? i doubt theres anywhere i could find liquid to swap it

**Reply by leoluk** [reply]
You can just send it back via Wormhole and get the original token back

**Reply by anonintern** [reply]
where does one send wormhole tokens?

**Reply by anonintern** [reply]
so when i go to send the WWT tokens back it gives me an approval error

**Reply by anonintern** [reply]
<img width="305" alt="Screen Shot 2021-08-10 at 10 58 14 AM" src="https://user-images.githubusercontent.com/88719086/128890333-b100dcd2-f3fa-4ee2-b610-ffb39005a339.png">

**Reply by leoluk** [reply]
Please contact the wormholebridge.com team: https://github.com/solana-labs/oyster/issues
